src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppManagement.java

```diff
@@ -0,0 +1,59 @@
+package com.hms_networks.americas.sc.extensions.system.application;
+
+import com.ewon.ewonitf.RuntimeControl;
+import com.hms_networks.americas.sc.extensions.fileutils.FileAccessManager;
+import com.hms_networks.americas.sc.extensions.logging.Logger;
+import java.io.IOException;
+
+/**
+ * Utility class for managing a Java application running on the Ewon Flexy system.
+ *
+ * @since 1.2.0
+ * @author HMS Networks, MU Americas Solution Center
+ */
+public class SCAppManagement {
+
+  /** File path for jvmrun file */
+  private static final String JVM_RUN_PATH = "usr/jvmrun";
```

Comment:
Should be updated to `/usr/jvmrun` to follow the established convention of using `/usr`